cmd/master-controller-manager/main.go

```diff
@@ -121,68 +114,24 @@ func main() {
 	// register the global error metric. Ensures that runtime.HandleError() increases the error metric
 	metrics.RegisterRuntimErrorMetricCounter("kubermatic_master_controller_manager", prometheus.DefaultRegisterer)
 
-	// register an operating system signals and context on which we will gracefully close the app
-	stopCh := signals.SetupSignalHandler()
```

Comment:
Who does signal handling now?

Reply:
it's handled by `controller-runtime` already

Reply:
https://github.com/kubermatic/kubermatic/pull/5683/files#diff-f4da825e24ae5359a2fb7393ede41246R173